frontend/src/pages/Profile.js

```diff
@@ -1,15 +1,303 @@
-import React from 'react';
+import React, {useContext, useEffect, useState} from 'react';
 import Alert from 'react-bootstrap/Alert'
+import Modal from "react-bootstrap/Modal";
+import ErrorMessage from "../messages/errorMessage";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import {AuthContext} from "../context/authContext";
+import Card from "react-bootstrap/Card";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import TogglePage from "../components/TogglePage";
+
 
 export default () => {
+    const {user, setUser} = useContext(AuthContext);
+    const [states, setStates] = useState([]);
```

Comment:
Let's change the name to "zones" or something like that. React has state word as well.